app/dashboard/migrations/0155_populate_profile_idena_token.py

```diff
@@ -0,0 +1,40 @@
+from django.db import migrations, transaction
+import uuid
+
+
+def gen_uuid_0(apps, schema_editor):
+    Profile = apps.get_model('dashboard', 'Profile')
+
+    existing_uuid = []
+
+    profiles_with_uuid = Profile.objects.filter(idena_token__isnull=False)
+    for profile in profiles_with_uuid:
+        idena_token = profile.brightid_uuid
+        print(f"UUID for profile -> {profile.pk} -> {idena_token}")
+        while idena_token in existing_uuid:
```

Comment:
Why using `while` here instead of `if`? Doesn't `idena_token in existing_uuid` just return a boolean?

Reply:
It can fail again on `if` when a duplicate UUID generated on line 15.
But I will refactor this migration and idena token logic.